### ISSUE TITLE
Failure for cron-cg-upload-delivery-report.sh

### ISSUE THREAD

**Issue by henrikstranneheim** [Clinical-Genomics/cg#1028]
## Description
Cron <hiseq.clinical@hasta> $CRONIC ${PRODUCTION_HOME}/servers/resources/hasta.scilifelab.se/crontabs/cron-cg-upload-delivery-report.sh >> ${LOG_BASE}/cg.upload-delivery-reports.log 2> >(tee -a ${LOG_BASE}/cg.upload-delivery-reports.log >&2)

Cronic detected failure for the command:
/home/proj/production/servers/resources/hasta.scilifelab.se/crontabs/cron-cg-upload-delivery-report.sh

```
----------------- DELIVERY_REPORT -------------
2021-03-12 15:21:30 hasta.scilifelab.se cg.cli.upload.delivery_report[3791] INFO Using analysis started at: 2021-03-11 13:40:58
2021-03-12 15:21:31 hasta.scilifelab.se cg.utils.commands[3791] INFO Running command /home/proj/bin/conda/envs/P_scout/bin/scout --config /home/proj/production/servers/config/hasta.scilifelab.se/scout.yaml export panel PEDHEP
2021-03-12 15:21:51 hasta.scilifelab.se cg.utils.commands[3791] INFO Running command /home/proj/bin/conda/envs/P_chanjo/bin/chanjo --config /home/proj/stage/servers/config/hasta.scilifelab.se/chanjo.yaml calculate coverage -s ACC6857A8 -f /home/hiseq.clinical/tmp/tmp4djtol6d
2021-03-12 15:21:59 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch latest version from bundle ablemutt
2021-03-12 15:21:59 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from bundle ablemutt
2021-03-12 15:21:59 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files with tags in [mip-config]
2021-03-12 15:21:59 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from version 33326
2021-03-12 15:21:59 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch latest version from bundle ablemutt
2021-03-12 15:21:59 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from bundle ablemutt
2021-03-12 15:21:59 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files with tags in [qcmetrics]
2021-03-12 15:21:59 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from version 33326
2021-03-12 15:22:00 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch latest version from bundle ablemutt
2021-03-12 15:22:00 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from bundle ablemutt
2021-03-12 15:22:00 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files with tags in [sampleinfo]
2021-03-12 15:22:00 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from version 33326
2021-03-12 15:22:00 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch version 2021-03-11 13:40:58 from bundle ablemutt
2021-03-12 15:22:00 hasta.scilifelab.se cg.cli.upload.delivery_report[3791] ERROR Unspecified error when uploading delivery report for case: ablemutt
2021-03-12 15:22:00 hasta.scilifelab.se cg.cli.upload.delivery_report[3791] INFO Uploading delivery report for case: cleverfalcon
----------------- DELIVERY_REPORT -------------
2021-03-12 15:22:00 hasta.scilifelab.se cg.cli.upload.delivery_report[3791] INFO Using analysis started at: 2021-03-11 00:16:02
2021-03-12 15:22:01 hasta.scilifelab.se cg.utils.commands[3791] INFO Running command /home/proj/bin/conda/envs/P_scout/bin/scout --config /home/proj/production/servers/config/hasta.scilifelab.se/scout.yaml export panel OMIM-AUTO
2021-03-12 15:22:21 hasta.scilifelab.se cg.utils.commands[3791] INFO Running command /home/proj/bin/conda/envs/P_chanjo/bin/chanjo --config /home/proj/stage/servers/config/hasta.scilifelab.se/chanjo.yaml calculate coverage -s ELH2292A1 -f /home/hiseq.clinical/tmp/tmpmrgkxhzh
2021-03-12 15:22:28 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch latest version from bundle cleverfalcon
2021-03-12 15:22:28 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from bundle cleverfalcon
2021-03-12 15:22:28 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files with tags in [mip-config]
2021-03-12 15:22:28 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from version 33134
2021-03-12 15:22:29 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch latest version from bundle cleverfalcon
2021-03-12 15:22:29 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from bundle cleverfalcon
2021-03-12 15:22:29 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files with tags in [qcmetrics]
2021-03-12 15:22:29 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from version 33134
2021-03-12 15:22:30 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch latest version from bundle cleverfalcon
2021-03-12 15:22:30 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from bundle cleverfalcon
2021-03-12 15:22:30 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files with tags in [sampleinfo]
2021-03-12 15:22:30 hasta.scilifelab.se housekeeper.store.api.find[3791] INFO Fetching files from version 33134
2021-03-12 15:22:30 hasta.scilifelab.se cg.apps.housekeeper.hk[3791] INFO Fetch version 2021-03-11 00:16:02 from bundle cleverfalcon
2021-03-12 15:22:30 hasta.scilifelab.se cg.cli.upload.delivery_report[3791] ERROR Unspecified error when uploading delivery report for case: cleverfalcon

RESULT CODE: 1
```

## This can be closed when

- [ ] When the crontab is working again

**Comment by henrikstranneheim** [comment]
@patrikgrenfeldt Could this be related to the PR https://github.com/Clinical-Genomics/cg/pull/653?

**Comment by patrikgrenfeldt** [comment]
resolved by #1029
